src/library.js

```diff
@@ -4047,24 +4046,25 @@ LibraryManager.library = {
 #if USE_PTHREADS
 // Pthreads need their clocks synchronized to the execution of the main thread, so give them a special form of the function.
                                "if (ENVIRONMENT_IS_PTHREAD) {\n" +
-                               "  _emscripten_get_now = function() { return performance['now']() - Module['__performance_now_clock_drift']; };\n" +
+                               "  return (function() { return performance['now']() - Module['__performance_now_clock_drift']; });\n" +
                                "} else " +
 #endif
 #if ENVIRONMENT_MAY_BE_SHELL
                                "if (typeof dateNow !== 'undefined') {\n" +
-                               "  _emscripten_get_now = dateNow;\n" +
+                               "  return dateNow;\n" +
                                "} else " +
 #endif
 #if MIN_IE_VERSION <= 9 || MIN_FIREFOX_VERSION <= 14 || MIN_CHROME_VERSION <= 23 || MIN_SAFARI_VERSION <= 80400 // https://caniuse.com/#feat=high-resolution-time
                                "if (typeof performance === 'object' && performance && typeof performance['now'] === 'function') {\n" +
-                               "  _emscripten_get_now = function() { return performance['now'](); };\n" +
+                               "  return performance['now'];\n" +
                                "} else {\n" +
-                               "  _emscripten_get_now = Date.now;\n" +
-                               "}",
+                               "  return Date.now;\n" +
+                               "}" +
 #else
                                // Modern environment where performance.now() is supported:
-                               "_emscripten_get_now = function() { return performance['now'](); };\n",
+                               "return performance['now'];\n" +
```

Comment:
do we still need quotations for `performance.now`? maybe with newer closure that is already recognized.